api/src/pcapi/sandboxes/scripts/creators/test_cases/__init__.py

```diff
@@ -0,0 +1,77 @@
+import random
+
+import factory.fuzzy
+
+from pcapi.core.offerers import factories as offerers_factories
+from pcapi.core.offerers import models as offerers_models
+from pcapi.core.offers import factories as offers_factories
+from pcapi.core.providers.titelive_gtl import GTLS
+
+
+def save_test_cases_sandbox() -> None:
+    create_offers_with_gtls()
+
+
+def create_offers_with_gtls() -> None:
+    venue = offerers_factories.VenueFactory(
+        name="Librairie des GTls",
+        venueTypeCode=offerers_models.VenueTypeCode.BOOKSTORE,
+        isPermanent=True,
+        latitude=45.91967,
+        longitude=3.06504,
+        address="13 AVENUE BARADUC",
+        postalCode="63140",
+        city="CHATEL-GUYON",
+        departementCode="63",
+    )
+    create_offers_for_each_gtl_level_1(10, venue)
+    create_offers_with_gtl_id("01030000", 10, venue)  # littérature, Œuvres classiques
+    create_offers_with_gtl_id("01030100", 10, venue)  # littérature, Œuvres classiques, Antiquité
+    create_offers_with_gtl_id(
+        "01030102", 10, venue
+    )  # littérature, Œuvres classiques, Antiquité, Littérature grecque antique
+
+    # un librairie que pour des mangas
+
+    venue2 = offerers_factories.VenueFactory(
+        name="Librairie des mangas",
+        venueTypeCode=offerers_models.VenueTypeCode.BOOKSTORE,
+        isPermanent=True,
+        latitude=46.66979,
+        longitude=-1.42979,
+        address="11 RUE GEORGES CLEMENCEAU",
+        postalCode="85000",
+        city="LA ROCHE-SUR-YON",
+        departementCode="85",
+    )
+    create_offers_with_gtl_id("03050300)", 10, venue2)  # 10 mangas
+
+
+def create_offers_for_each_gtl_level_1(size_per_gtl_level_1: int, venue: offerers_models.Venue) -> None:
+    for gtl_id_prefix in range(1, 14):
+        for _ in range(size_per_gtl_level_1):
+            gtl_id = random.choice(
+                list(
+                    filter(
+                        lambda gtl_id, gtl_id_prefix=gtl_id_prefix: gtl_id.startswith(str(gtl_id_prefix).zfill(2)),  # type: ignore [arg-type]
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏Là on maitrise toute la fonction, c'est pas normal d'avoir un `ignore`